anvio/anvi_thread.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+"""A lightly modified Thread sublcass.  Tracks the return values of Thread's target."""
+
```

Comment:
I think it would be much better to call this `anvio/threadingops.py` so we can expand it with more classes and functions :)